src/docfx/lib/html/HtmlToken.cs

```diff
@@ -67,5 +67,18 @@ public void SetAttributeValue(string name, string value)
             Attributes.CopyTo(attributes);
             Attributes = attributes;
         }
+
+        public string GetAttributeValueByName(string name)
+        {
+            foreach (ref var attribute in Attributes.Span)
+            {
+                if (attribute.NameIs(name))
+                {
+                    return attribute.Value.ToString();
+                }
+            }
+
+            return "";
```

Comment:
return `null` in case the attribute is missing
```suggestion
            return null;
```